### ISSUE TITLE
Error received from starting extension host (kind: LocalProcess)

### ISSUE THREAD

**Issue by fantasyzhjk** [microsoft/vscode#165125]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 
Version: 1.72.2
Commit: d045a5eda657f4d7b676dedbfa7aab8207f8a075
Date: 2022-10-12T22:15:18.074Z
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22622
Sandboxed: No

## Steps to Reproduce:

1. Open a project in vscode
2. Console shows error

## Log

```
[2022-11-01 14:54:46.668] [renderer1] [error] Error received from starting extension host (kind: LocalProcess)
[2022-11-01 14:54:46.669] [renderer1] [error] ["Error: Cannot load entry script from outisde the application.",
"    at new UtilityProcess (node:electron/js2c/browser_init:153:1206)",
"    at a.start (D:\\scoop\\apps\\vscode\\1.72.2\\resources\\app\\out\\vs\\code\\electron-main\\main.js:80:50098)",
"    at Ut.start (D:\\scoop\\apps\\vscode\\1.72.2\\resources\\app\\out\\vs\\code\\electron-main\\main.js:80:46042)",
"    at Object.call (D:\\scoop\\apps\\vscode\\1.72.2\\resources\\app\\out\\vs\\code\\electron-main\\main.js:24:8334)",
"    at l.onPromise (D:\\scoop\\apps\\vscode\\1.72.2\\resources\\app\\out\\vs\\code\\electron-main\\main.js:23:5812)",
"    at l.onRawMessage (D:\\scoop\\apps\\vscode\\1.72.2\\resources\\app\\out\\vs\\code\\electron-main\\main.js:23:5207)",
"    at D:\\scoop\\apps\\vscode\\1.72.2\\resources\\app\\out\\vs\\code\\electron-main\\main.js:23:4499",
"    at y.invoke (D:\\scoop\\apps\\vscode\\1.72.2\\resources\\app\\out\\vs\\code\\electron-main\\main.js:19:145)",
"    at S.deliver (D:\\scoop\\apps\\vscode\\1.72.2\\resources\\app\\out\\vs\\code\\electron-main\\main.js:19:2269)",
"    at C.fire (D:\\scoop\\apps\\vscode\\1.72.2\\resources\\app\\out\\vs\\code\\electron-main\\main.js:19:1847)",
"    at D:\\scoop\\apps\\vscode\\1.72.2\\resources\\app\\out\\vs\\code\\electron-main\\main.js:17:19380",
"    at y.invoke (D:\\scoop\\apps\\vscode\\1.72.2\\resources\\app\\out\\vs\\code\\electron-main\\main.js:19:145)",
"    at S.deliver (D:\\scoop\\apps\\vscode\\1.72.2\\resources\\app\\out\\vs\\code\\electron-main\\main.js:19:2269)",
"    at C.fire (D:\\scoop\\apps\\vscode\\1.72.2\\resources\\app\\out\\vs\\code\\electron-main\\main.js:19:1847)",
"    at D:\\scoop\\apps\\vscode\\1.72.2\\resources\\app\\out\\vs\\code\\electron-main\\main.js:17:19564",
"    at y.invoke (D:\\scoop\\apps\\vscode\\1.72.2\\resources\\app\\out\\vs\\code\\electron-main\\main.js:19:145)",
"    at S.deliver (D:\\scoop\\apps\\vscode\\1.72.2\\resources\\app\\out\\vs\\code\\electron-main\\main.js:19:2269)",
"    at C.fire (D:\\scoop\\apps\\vscode\\1.72.2\\resources\\app\\out\\vs\\code\\electron-main\\main.js:19:1847)",
"    at Y (D:\\scoop\\apps\\vscode\\1.72.2\\resources\\app\\out\\vs\\code\\electron-main\\main.js:17:21502)",
"    at IpcMainImpl.u (D:\\scoop\\apps\\vscode\\1.72.2\\resources\\app\\out\\vs\\code\\electron-main\\main.js:16:50233)",
"    at IpcMainImpl.emit (node:events:526:28)",
"    at EventEmitter.<anonymous> (node:electron/js2c/browser_init:169:11345)",
"    at EventEmitter.emit (node:events:526:28)"]
```

**Comment by alexdima** [comment]
@fantasyzhjk Does configuring `"extensions.experimental.useUtilityProcess": false` make any difference for you? Does this reproduce all the time? Do you have a special setup where you have installed VS Code in a special location on disk?

fyi @digitarald

**Comment by deepak1556** [comment]
/duplicate https://github.com/microsoft/vscode/issues/162533

This is fixed in Insiders and will be available with `1.74` stable.
